wordpress-mobile/WordPress-iOS
# ULS: Display navigation bar large titles

Ref: https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/issues/182
Auth PR: https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/307

This uses the Auth branch to display Large Titles in the navigation bar. 

As noted in the Auth PR, displaying the large titles in the email login flow is for testing purposes only, and will be removed before the PRs are merged.

To test:

---
- Run the app. Log out if necessary.
- `Log In` > `Continue with WordPress.com`.
- Verify:
  - The back button label is 'Back'.
  - `Log In` is displayed in the nav bar.

![login_email](https://user-images.githubusercontent.com/1816888/84959210-2471cf00-b0bc-11ea-8520-aa0ba64385fb.png)

---
- Enter an email address and `Next`.
- Verify:
  - The back button label is 'Back'.
  - `Log In` is displayed in the nav bar.

![login_magic_link](https://user-images.githubusercontent.com/1816888/84959497-aa8e1580-b0bc-11ea-97dd-f94a08efb50d.png)

---
- Select `Enter your password instead.`
- Verify:
  - The back button label is 'Back'.
  - There is no large title displayed in the nav bar.

![login_password](https://user-images.githubusercontent.com/1816888/84959509-afeb6000-b0bc-11ea-8a3d-6abaf5d0dca7.png)

---
- Go into any other login or signup flow.
- Verify large titles are not displayed in the nav bar.

---
PR submission checklist:
